packages/infra/lib/api-stack.ts

```diff
@@ -444,24 +445,53 @@ export class APIStack extends Stack {
     //-------------------------------------------
     // Surescripts
     //-------------------------------------------
-    const {
-      synchronizeSftpLambda: surescriptsSynchronizeSftpLambda,
-      synchronizeSftpQueue: surescriptsSynchronizeSftpQueue,
-      receiveFlatFileResponseLambda: surescriptsReceiveFlatFileResponseLambda,
-      receiveVerificationResponseLambda: surescriptsReceiveVerificationResponseLambda,
-      sendPatientRequestLambda: surescriptsSendPatientRequestLambda,
-      sendPatientRequestQueue: surescriptsSendPatientRequestQueue,
-      receiveFlatFileResponseQueue: surescriptsReceiveFlatFileResponseQueue,
-      receiveVerificationResponseQueue: surescriptsReceiveVerificationResponseQueue,
-      surescriptsReplicaBucket,
-      medicationBundleBucket,
-    } = new SurescriptsNestedStack(this, "SurescriptsNestedStack", {
-      config: props.config,
-      vpc: this.vpc,
-      alarmAction: slackNotification?.alarmAction,
-      lambdaLayers,
+    let surescriptsSynchronizeSftpLambda: Lambda | undefined;
+    let surescriptsSynchronizeSftpQueue: Queue | undefined;
+    let surescriptsReceiveFlatFileResponseLambda: Lambda | undefined;
+    let surescriptsReceiveVerificationResponseLambda: Lambda | undefined;
+    let surescriptsSendPatientRequestLambda: Lambda | undefined;
+    let surescriptsSendPatientRequestQueue: Queue | undefined;
+    let surescriptsReceiveFlatFileResponseQueue: Queue | undefined;
+    let surescriptsReceiveVerificationResponseQueue: Queue | undefined;
+    let surescriptsReplicaBucket: s3.Bucket | undefined;
+
+    const medicationBundleBucket = new s3.Bucket(this, "MedicationBundleBucket", {
+      bucketName: props.config.medicationBundleBucketName,
+      publicReadAccess: false,
+      encryption: s3.BucketEncryption.S3_MANAGED,
+      versioned: true,
     });
 
+    if (!isSandbox(props.config)) {
```

Comment:
Instead of checking the environment, can we rely on the presence of `props.config.surescripts`? Then we have a single source or truth for creating the nested stack or not